src/components/course/data/constants.js

```diff
@@ -88,7 +93,8 @@ export const REASON_USER_MESSAGES = {
 };
 
 export const DISABLED_ENROLL_REASON_TYPES = {
-  POLICY_NOT_ACTIVE: 'policy_not_active',
+  SUBSIDY_NOT_ACTIVE: 'subsidy_expired',
```

Comment:
[clarification] Similar question as above. Are `SUBSIDY_NOT_ACTIVE`, `POLICY_NOT_ACTIVE`, and `REASON_POLICY_SPEND_LIMIT_REACHED` used by the MFE, or are these returned by the `can-redeem` API but otherwise unused?

Reply:
The reason types do need to remain since that is the only coupling between the api and the rendering component to verify the correct error is being displayed